pennylane/pauli/pauli_arithmetic.py

```diff
@@ -116,6 +116,15 @@ def __init__(self, mapping):
                 del mapping[wire]
         super().__init__(mapping)
 
+    def __reduce__(self):
+        """Defining this method allows PauliWord to be pickle-able. Otherwise, un-pickling
+        would cause __setitem__ to be called, which is forbidden on PauliWord.
+
+        For more information, see: https://docs.python.org/3/library/pickle.html#object.__reduce__
+
+        """
```

Comment:
```suggestion
        """Defines how to pickle and unpickle a PauliWord. Otherwise, un-pickling
        would cause __setitem__ to be called, which is forbidden on PauliWord.
        
        For more information, see: https://docs.python.org/3/library/pickle.html#object.__reduce__
        """
```